R/expand_model_out_grid.R

```diff
@@ -345,6 +378,22 @@ null_taskids_to_na <- function(model_task) {
   )
 }
 
+# Set derived task_ids to all NULL values.
+derived_taskids_to_na <- function(model_task, derived_task_ids) {
+  if (!is.null(derived_task_ids) || length(derived_task_ids) > 0L) {
```

Comment:
Did you want an `&&` in the condition here?  Again, I am just not sure what the inputs here might look like to be sure.  I could imagine `derived_task_ids = character(0)` passing this condition but being a surprise?

Reply:
IHmmm you're right, a `&&` statement would be more appropriate there but in all honesty, given the validation of `derived_task_ids` returns either a vector of valid task ID names or `NULL`, the second check is not necessary. Removed in c6c54237d22cdffc18d5eca811fd7331cd6c698b